src/graph.rs

```diff
@@ -212,16 +219,17 @@ fn trace_branch<'repo>(
     branch_index: usize,
 ) -> Result<bool, Error> {
     let mut curr_oid = oid;
-    let start_index;
+    let start_index: i32;
     let mut any_assigned = false;
     loop {
         let index = indices[&curr_oid];
         let info = &mut commits[index];
         if info.branch_trace.is_some() {
-            if index > 0 {
-                start_index = index - 1;
+            // TODO: remove exception (see TODO below)
+            if index > 0 || !any_assigned {
```

Comment:
@adamreichold Currently, I use the fact that still existing branches (i.e not derived from merge summary, thus still existent and not filtered out) can be identified by there "start" index being smaller than their "end" index (index reverse to chronology!), and setting there range to (None, None) to identify them later (when assigning the column to draw them).

Would you say that is ok, or would you introduce a state variable like `any_assigned`?